### PR TITLE
Changes Everipedia to BrainDAO

### PR DESCRIPTION
**Pull request by Royal-lobster**
# Changes
- changes Everipedia text to BrainDAO where nessasary

### CODE REVIEW

File: src/components/SEO/Default.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const SEOHeader = ({ router }: SEOHeaderProps) => (
     twitter={{
       cardType: 'summary_large_image',
       handle: '@Everipedia',
-      site: 'Everipedia',
+      site: 'BrainDao',
```
Comment (Royal-lobster) [inline src/components/SEO/Default.tsx:31]:
```suggestion
      site: 'BrainDAO',
```

**Review by Softdev1 — APPROVED** [review]
This is cool now @Royal-lobster

File: src/components/SEO/Default.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ const SEOHeader = ({ router }: SEOHeaderProps) => (
     twitter={{
       cardType: 'summary_large_image',
       handle: '@Everipedia',
-      site: 'Everipedia',
+      site: 'BrainDAO',
```
Comment (kesar) [inline src/components/SEO/Default.tsx:31]: https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary

site btw should be a twitter handle, so probably @everipedia as well

Comment (Royal-lobster) [reply src/components/SEO/Default.tsx:31]: wow 😲

Comment (kesar) [reply src/components/SEO/Default.tsx:31]: old was wrong too I think it needs the @ (but didnt test it)

**Pull request merged**